test/libdnf/transaction/TransformerTest.cpp

```diff
@@ -0,0 +1,216 @@
+#ifdef SKIP
```

Comment:
Plerase could you explain why it is marked as skip?

Reply:
Explained, added TODO.